generators/generator-base.js

```diff
@@ -472,6 +472,7 @@ module.exports = class extends PrivateBase {
             { name: 'Greek', value: 'el' },
             { name: 'Hindi', value: 'hi' },
             { name: 'Hungarian', value: 'hu' },
+            { name: 'Bahasa Indonesia', value: 'id' },
```

Comment:
Anyway I have changed it here already. In the UI however it will be bahasa Indonesia